bq-metrics-extractor/config/config.yaml

```diff
@@ -19,10 +19,23 @@ bqmetrics:
         WHERE event_name = "first_open"
         LIMIT 1000
   - type: summary
+    name: sims_who_have_used_data
+    help: Number of SIMs that has used data last 24 hours
+    resultColumn: count
+    sql: >
+        SELECT count(DISTINCT msisdn) FROM `pantel-2decb.data_consumption.raw_consumption`
```

Comment:
don't we need to name the result  ? e.g. `SELECT count(DISTINCT msisdn) as count`